python_bindings/correctness/basics.py

```diff
@@ -153,6 +153,17 @@ def test_basics3():
     left += 5
     left += ss
 
+def test_basics4():
+    x = hl.Var('x')
```

Comment:
Should probably add a comment about what this is testing for (and perhaps a link to the GitHub issue) for future reference

Reply:
Fixed.